CONTRIBUTING.md

```diff
@@ -1,55 +1,69 @@
+we welcome bug reports and feature requests. Please search the [issue tracker](https://github.com/OpenRefine/OpenRefine/issues) first to make sure
+the bug / feature hasn't already been added. Note: the development team principally works from the issue
+tracker, so anything not included there risks getting lost.
 
 ## Promote OpenRefine
 
-You don't need to be a coder to contribute to OpenRefine. Did you write a tutorial or article about OpenRefine on your blog or site? Are you organizing a workshop or presentation for OpenRefine in your city? Let us know via our [forum](https://forum.openrefine.org/) or Twitter account ([@OpenRefine](http://twitter.com/OpenRefine)). We will share the news via our monthly update and via our Twitter handle. 
+Promoting OpenRefine is a great way to give back. Did you write a tutorial or article about OpenRefine on your blog or site?
+Are you organizing a workshop or presentation for OpenRefine in your city? Let us know via our [forum](https://forum.openrefine.org/) or Twitter account ([@OpenRefine](http://twitter.com/OpenRefine)).
+We will share the news via our monthly update and via our Twitter handle.
 
-## Contributing translations
+## Contribute translations
 
-You can help us [translate OpenRefine](https://docs.openrefine.org/technical-reference/translating-ui) in as many languages as possible [via Weblate](https://hosted.weblate.org/engage/openrefine/?utm_source=widget).
+We want OpenRefine to be available in as many languages as possible to serve
+the biggest community of users. You can help us [translate OpenRefine](https://docs.openrefine.org/technical-reference/translating-ui) into languages you are fluent in [via Weblate](https://hosted.weblate.org/engage/openrefine/?utm_source=widget).
+Although we have the beginnings of translations for many languages, only a few are complete and popular languages
+like Spanish, Brazilian Portuguese, and French could use help.
 
-## Contributing documentation
+## Contribute documentation
 
-When browsing our documentation, feel free to use the edit button to suggest improvements. For large changes, you might want to [prepare your changes locally](https://openrefine.org/docs/technical-reference/contributing#contributing-to-the-documentation).
+When browsing our [user manual](https://openrefine.org/docs/) or other documentation, feel free to use the edit button to suggest improvements. For large changes, you might want to [prepare your changes locally](https://openrefine.org/docs/technical-reference/documentation-contributions).
 
-##  Contributing code 
+##  Contribute code 
 
 You can contribute code in various ways:
 - Fix bugs or implement new features. Follow [our guide towards your first code contribution](https://openrefine.org/docs/technical-reference/code-contributions)
+- Improve test coverage. Much of our code was originally written without tests, so help on this front is very much appreciated.
 - Develop an OpenRefine extension
```

Comment:
Should we remove since it is redundant with the following section `New functionality via extensions`

Reply:
I think these are two different things. We insist on test coverage for new functionality and bug fixes, but we'd also like to improve test coverage for our legacy code. As an example, Apply Operation History currently has basically zero test coverage, yet people promote it relentlessly. If it's going to be promoted, it should have tests written for it.